Tests/Runner/StateFileTest.php

```diff
@@ -111,7 +111,6 @@ public function testCreateStateFileFromStateWithoutNamespace()
         $fileName = $this->dataDir . DIRECTORY_SEPARATOR . 'in' . DIRECTORY_SEPARATOR . 'state.json';
         self::assertTrue(file_exists($fileName));
         $obj = new \stdClass();
-        $obj->lastUpdate = 'today';
         self::assertEquals(
```

Comment:
dtto - test by se mel smazat, anebo aspon prejmenovat testCreateStateFileFromInvalidState

ale zvazil bych jestli je to zadouci chovani - me prijde, ze by to melo spis zarvat

Reply:
jak bys ho pojmenoval? a mělo by to zařvat, že to nemá správnou strukturu? možná jo, ale v separátním PR?

Reply:
tvl dyt pisu `testCreateStateFileFromInvalidState` :)

Reply:
nevim jestli to delat v separatnim PR - ted se to chova tak, ze kdyz ve state je neco, co neni namepsacovany, tak se to tvari jako by tam nic nebylo - to me prijde dost obskurni chovani

Reply:
oukej!

Reply:
přejenoval jsem aktuální `Configuration\State` na `Configuration\ComponentState` a přidal novej `State`